### PR TITLE
List events in readme

### PR DESCRIPTION
**Pull request by naufraghi**

### CODE REVIEW

**Comment by naufraghi** [comment]
Issue links are not working...

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # meetings
-Our meetings
+
+Our meetings:
+
+ - December 20 TBA
+ - November 22 rust-torino/meetings#3
```
Comment (lu-zero) [inline README.md:6]:
```suggestion
 - [November 22](https://github.com/rust-torino/meetings/issues/3)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+ - October 25 rust-torino/meetings#2
```
Comment (lu-zero) [inline README.md:7]:
```suggestion
 - [October 25](https://github.com/rust-torino/meetings/issues/2)
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
...
+ - September 28 rust-torino/meetings#1
```
Comment (lu-zero) [inline README.md:8]:
```suggestion
 - [September 28](https://github.com/rust-torino/meetings/issues/1)
```

**Pull request merged**